docs/language/operators/from.md

```diff
@@ -54,20 +66,105 @@ from ... | switch color (
 ) | ...
 ```
 
-The output of a fork consists of multiple legs that must be merged.
-If the downstream operator expects a single input, then the output legs are
-merged with an automatically inserted [combine operator](combine.md).
+### Input Data
+
+Examples below below assume the existence of the Zed lake created and populated
+by the following commands:
+
+```mdtest-command
+export ZED_LAKE=example
+zed -q init
+zed -q create coinflips
+echo '{flip:1,result:"heads"} {flip:2,result:"tails"}' | zed -q -use coinflips load -
+zed -q -use coinflips branch trial 
+echo '{flip:3,result:"heads"}' | zed -q -use coinflips@trial load -
+zed -q create numbers
+echo '{number:1,word:"one"} {number:2,word:"two"} {number:3,word:"three"}' | zed -q -use numbers load -
+zed ls | awk '{ print $1 }'
+```
+
+The lake then contains the two pools:
+
+```mdtest-output
+coinflips
+numbers
+```
```

Comment:
I actually didn't want to have to include any `mdtest-output` here, but it seems like in order to have an `mdtest-command` (such as I needed above to prep the Zed lake needed for the examples) I'd need to have a matching `mdtest-output`. Therefore I figured showing the pool names could make it happy while being brief & informative enough for the reader to not be seen as absolute clutter. 🤷‍♂️ 

It'd be cool if someone was willing to add some kind of `mdtest-command` option that says no matching `mdtest-output` is expected, but I'm happy to live without it for now.